relays/list_transform.ts

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+import { Message } from "mhub";
+
+/**
+ * @param msg Message just received on this binding.
+ * @return Message(s), promise for message(s), or void
+ */
+export default function(msg: Message): void|Message|Message[]|Promise<Message> {
+	if (msg.topic === "scores:ranking") {

```

Comment:
If you put the right filter (pattern) in the binding config, you don't need to check for `msg.topic` here, and can simply process all incoming messages. It can still be useful to keep it, though, but that would be more for documentation purposes then.